app/views/static/index.html.erb

```diff
@@ -44,9 +44,9 @@
         <div class="flex flex-col justify-center items-center mx-auto lg:mx-0 text-center lg:text-left">
           <div class="text-4xl">
             <div>
-              <h2 class="">The easiest and most love-filled <br/> way to manage your <br/> Human Essentials bank.</h2>
+              <h2 class="">The easiest and most love-filled <br> way to manage your <br> Human Essentials bank.</h2>
               <p class='text-lg font-normal mb-3 mt-2'>Ready to make yourself more efficient?</p>
-              <a href=<%="#{new_account_request_path}"%>><button type="button" class="text-2xl bg-green-600 text-white px-3 py-2 rounded-2xl hover:bg-green-700">
+              <a href="<%= "#{new_account_request_path}" %>"><button type="button" class="text-2xl bg-green-600 text-white px-3 py-2 rounded-2xl hover:bg-green-700">
```

Comment:
This looks like a mistake with the quotes.

Reply:
These are intentional. The linter says "expected attribute value after '=' (at >)". Which I believe means that it wants every attribute in an tag to be in the form `<tag attr="<%= some_stuff %>">`. 

I think they parse the same, but its the same logic as disallowing `<% #These types of comments %>`. vs `<%# allowing these %>`

Reply:
I don't mean the linter made a mistake, I mean the human did. :slightly\_smiling\_face: There's no need to have the inside of the ERB be a string with nothing but a single interpolation.

Reply:
I get what you are saying now. The quotes in the erb not in the html.